pkg/config/virtio.go

```diff
@@ -778,6 +819,10 @@ func (nbd *NetworkBlockDevice) FromOptions(options []option) error {
 	return nbd.NetworkBlockStorageConfig.FromOptions(unhandledOpts)
 }
 
+func (nbd *NetworkBlockDevice) Validate() error {
```

Comment:
Since `NetworkBlockDevice` embeds a `DiskStorageConfig`, it  might be possible to omit this one.

Reply:
We cannot omit this one as `NetworkBlockDevice` has other fields also:

```
type NetworkBlockDevice struct {
	NetworkBlockStorageConfig
	DeviceIdentifier    string
	Timeout             time.Duration
	SynchronizationMode NBDSynchronizationMode
}
```